### DISCUSSION TITLE
img2img prompt bleeding over to text2img

### DISCUSSION THREAD

**Discussion by jan1979x** [vladmandic/sdnext#1856] [General]
thank you for your hard work Vlad
i am on the latest version [Version: 7b35f78d Thu Jul 27 20:07:27]
txt2img has a prompt, img2img a different prompt (with lora's i might add)

after using both and going back to text2img, parts (prompts) of the img2img prompt are showing in the results of text2img 
like its bleeding over somehow

**Comment by jan1979x** [comment]
one time happening its minor ill close this

**Comment by josh5671** [comment]
Hmm. i literally same problem.
But as i not sure how to describe, this problem stay me here for almost a week+- now. .only able to fix it by restarting the server, and problem coming back when you img2img again.

Recent update cause the txt2img produce small blackbox on result. 'fixed' the bleed but grow another issue. 


PS:  no error on my side, so I nothing to post for feedback.

**Reply by jan1979x** [reply]
Hi, ok i thought it was me, i do see if you use a controlnet in img2img and leave it turned on, it uses it in text2img at times

**Reply by josh5671** [reply]
I may suggest you open a ticket in the issue section. for myself..i rarely use img2img not even inpainting, only for upscale.

**Comment by vladmandic** [comment]
i have an idea what it might be, best to open a issue so i can track it (cant take a look immediately as i'm traveling)
